soccer/gameplay/skills/capture.py

```diff
@@ -101,11 +105,16 @@ def execute_course_approach(self):
         # don't hit the ball on accident
         self.robot.set_avoid_ball_radius(Capture.CourseApproachAvoidBall)
         pos = self.find_intercept_point()
-        self.robot.face(main.ball().pos)
+
+        if (self.faceBall):
```

Comment:
The facing logic may be better to put in `execute_running()` since it applies to all substates of `running`.